elasticsearch/README.md

```diff
@@ -35,6 +35,21 @@ Example:
 ```
 $ oc exec -c elasticsearch $POD -- allocate-stale-primary .kibana.02c55f18a892b365bcd1802db9e5c9df39c04674
 ```
+### export-kibana-objects
+Export kibana objects for a given user from their kibana index as a JSON search response
+
+Example:
+```
+$ oc exec -c elasticsearch $POD -- export-kibana-objects admin2
+```
+### import-kibana-objects
+Import kibana objects from a JSON search response produced by `export-kibana-objects` to the kibana index in the
+input file.
+
+Example:
+```
+$ oc exec -i -c elasticsearch $POD -- import-kibana-objects kib.json
```

Comment:
should this be
```
cat kib.json | oc exec -i -c elasticsearch $POD -- import-kibana-objects
```
?

Reply:
no.  the `-i` allows you to stream `CMD="oc exec -i -c elasticsearch $pod -- es_util" ./import-kibana-objects /home/centos/cluster-logging-tools/scripts/kib.json`

Reply:
I don't understand.  So `import-kibana-objects` can be used both inside and outside of the container?

Reply:
Updated and tested script reads from piped, stdin, updated README